domains/_github-pages-challenge-Arch881010.arch.json

```diff
@@ -1,13 +1,17 @@
 {
   "description": "My personal website on arch.is-a.dev (main files in repo/repository)[Github Challenge].",
   "repo": "https://github.com/Arch881010/Arch881010",
-  "owner": {
+  "owner": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJkYXRhIjp7InVzZXJfaWQiOiIxMTE3ODkzMTAiLCJ1c2VybmFtZSI6IkFyY2g4ODEwMTAiLCJlbWFpbCI6ImFyY2gxMDEwQGljbG91ZC5jb20ifSwiZXhwaXJlc0luIjoiOTk5OXkiLCJpYXQiOjE3MjE4NzIxNzB9.UIfmbJXRHNXMi1l3P5hYx6d4IdunEjxUiXDU_XwZHJc",
+  "note": "This is the ID from owl (https://owl.is-a.dev/id/111789310/Arch881010)",
```

Comment:
Owl ID is not complete and is going to be changing to a different encryption method soon.